data/core/editor/help.cfg

```diff
@@ -122,151 +122,208 @@ Named locations can be accessed from WML using the Standard Location Filter’s
 # wmllint: markcheck on
 
 # wmllint: markcheck off
+# wmllint: unbalanced-on
 [topic]
     id=editor_tool_label
     title= _ "Label Tool"
-    text= "<img>src=icons/action/editor-tool-label_60.png align=left box=yes</img>" + _"TODO"
+    text= "<img>src=icons/action/editor-tool-label_60.png align=left box=yes</img>" + _ "Put text labels on the map.
+
+• Left-click will open a dialog box to create a new label or edit an existing one.
+• Right-click deletes.
+• Drag-and-drop with the left mouse button moves labels.
+
+This tool is only available in Scenario Mode, the decorations are implemented in the scenario using WML’s <italic>text='[label]'</italic> tag."
 [/topic]
+# wmllint: unbalanced-off
 # wmllint: markcheck on
 
 # wmllint: markcheck off
+# wmllint: unbalanced-on
 [topic]
-    id=editor_tool_item
-    title= _ "Item Tool"
-    text= "<img>src=icons/action/editor-tool-item_60.png align=left box=yes</img>" + _"TODO"
+    id=editor_tool_scenery
+    title= _ "Item Tool (Scenery Tool)"
+    text= "<img>src=icons/action/editor-tool-item_60.png align=left box=yes</img>" + _ "The Item Tool allows placing decorations such as windmills, bookcases and monoliths. Multiple items can be placed on the same hex.
+
+<bold>text='Note:'</bold> the tool doesn’t support deleting items once placed, nor does it support undo. Mistakes can currently only be fixed by editing the generated WML file.
+
+This tool is only available in Scenario Mode, the decorations are not part of the terrain and are implemented in the scenario using WML’s <italic>text='[item]'</italic> tag."
 [/topic]
+# wmllint: unbalanced-off
 # wmllint: markcheck on
 
 # wmllint: markcheck off
+# wmllint: unbalanced-on
 [topic]
     id=editor_tool_village
     title= _ "Village Ownership Tool"
-    text= "<img>src=icons/action/editor-tool-village_60.png align=left box=yes</img>" + _"TODO"
+    text= "<img>src=icons/action/editor-tool-village_60.png align=left box=yes</img>" + _ "This tool assigns ownership of villages at the start of a scenario. The villages must first be placed on the terrain with the <ref>dst='editor_tool_paint' text='Paint Tool'</ref>.
+
+• Left-click will assign the village to the currently-selected side.
+• Right-click will set the village back to unowned.
+
+This tool is only available in Scenario Mode, ownership information is implemented using WML’s <italic>text='[side][village]'</italic> tag."
```

Comment:
Should be a semicolon instead of a comma.

Reply:
Also those are two tags, so "tag" should be plural.

Reply:
Ended up rewriting it as `This tool is only available in Scenario Mode; ownership information is stored by adding WML <italic>text='[village]'</italic> tags to the appropriate <italic>text='[side]'</italic>."`

If this is the biggest change needed then I'm happy.